README.md

```diff
@@ -1,8 +1,16 @@
 # Synergistic Software Platform for AI, Physics Simulations, and Experiments (Synapse)
 
+Synapse is a modular framework to build components of digital twins.
+It supports operators of machines and experiments with ML-assisted predictions, trained on a combination of continously measured and simulated data.
+Synapse embraces emerging [integrated research infrastructures](https://www.nersc.gov/what-we-do/computing-for-science/integrated-research-infrastructure), deploying a user-facing cloud service, using HPC/cloud compute, and exchanging modular components via container registries.
+
+At the moment, Synapse uses the NERSC Spin (control, dashboard), NERSC SuperFacilityAPI (simulation submission & ML training on PM), and NERSC container registry.
+Synapse is actively developed and broadened to an AI-accelerated, portable framework as part of the [DOE Genesis Mission](https://genesis.energy.gov).
```

Comment:
Hm, we are not yet officially part of Genesis, no? (In the sense that there has not yet even been a call for proposal for Genesis.)